go.mod

```diff
@@ -30,11 +30,11 @@ require (
 	github.com/prometheus/client_golang v1.17.0
 	github.com/prometheus/client_model v0.5.0
 	github.com/rook/rook v1.12.0-alpha.0
-	github.com/rook/rook/pkg/apis v0.0.0-20231201141116-eacc7e74412e
+	github.com/rook/rook/pkg/apis v0.0.0-20231211195439-c80ea7b64424
```

Comment:
can we just update rook dependency for now, as it is making it complicated

Reply:
The other updates are part of the rook api updates itself, I have not updated them manually.